src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 0;
+
+  const getSum = function(num1) {
+    count++;
+
+    return function(num2) {
+      if (count > 3 && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return num1 + num2;
+      }
```

Comment:
no needs to add `else` if you use early return at `if`
```suggestion
      if (count > 3 && count % 2 === 0) {
        return 'Bzzz... Error!';
      }
        
      return num1 + num2;
      
```